.gitignore

```diff
@@ -1,10 +1,15 @@
 # Python stuff
 *.py[cdo]
+.venv
 
 # Editor detritus
 *.vim
 *.swp
 tags
+# If you wish to contribute and use a different editor, please don't 
+# extend this list, or it will quickly get very long.
+# Instead create a '.gitignore-global' file in your home directory and 
+# run 'git config --global core.excludesfile ~/.gitignore-global'
```

Comment:
Personally, I don't think that gpiozero has so many contributors that this would become a problem :wink:

Reply:
😞 it's such an awesome project with a great purpose and really nice code.
There must be more dads out there who, like me, stumble across it with their kids and want to learn/hone skills and help out...

Reply:
I agree that it's a great project (which, as I'm sure you've discovered by now is a lot more complex under-the-hood than it first appears), but IMHO telling people that they need to set up a `.gitignore-global` is a bit of a needless barrier to any potential contributors.

Reply:
@lurch - that's interesting feedback. Personally, I found the changes I needed to make to .gitignore etc. were the biggest barriers, as I felt very uncomfortable doing that for someone else's project. Each to their own, I guess.
How does this comment look. I've updated it here (because of this conversation) and also in #1122 where it belongs.

Reply:
@lurch  - I've found your opinions to be very valuable guidance.

and yes, life would be a lot easier for me too, if I were able to keep them separate. Doing that in this case gives me a few bad options: 1) manage 4992 untracked files in my repo and run no tests in my github pipeline (too error prone for my liking); 2) do some history-rewriting, rebasing magic just before creating the PR (again error prone); 3) wait a few months for the first PR to be reviewed and approved before doing anything else (not really very attractive either)

Again, thanks for your guidance and ideas. I'm going to move on for now and leave this PR open in case a fix for this bug is wanted at some point in the future ...